lib/internal_metric/global.rb

```diff
@@ -26,7 +26,8 @@ class Global < Base
               :scheduled_jobs_stuck,
               :missing_s3_uploads,
               :version_info,
-              :readonly_sites
+              :readonly_sites,
+              :pg_highest_seq
```

Comment:
```suggestion
              :postgres_highest_sequence
```

All the other postgres related metrics are prefixed with `postgres` so I think we should go with the existing convention instead of `pg`. I also think we can just refer to the metric as `sequence` instead of `seq` since I think there is very little value in shortening the word.

Reply:
:+1: done